docs/rst/manual/developer_documentation/dev-strategy.rst

```diff
@@ -88,59 +88,55 @@ This section contains a list of epics suggested to be carried out to continue
-monitor usage of ERT and act upon it.
-
-Implement a backend of the storage API using EnKFMain + file storage
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-To utilise strangulation techniques and to make the new visualisation solution
-available to a larger set of users we intend to implement the storage API using
-EnKFMain and file storage. It will rely heavily on shared disk access to user
-configuration and will loose all data if the storage files are deleted from
-disk. All in all, is should behave similarly to what EnKFMain and file storage
-does today.
-
-Make all data exposed to users pass through the storage API
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Make all data exposed to users pass through the storage API [in progress]
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 With an implementation of the storage API backed on EnKFMain and file storage
 we are again ready to aim for all user facing data (visualisation and export)
-to pass through the storage API.
+to pass through the storage API. Ensure that this can be done while ERT is
+running. Deprecate the collectors so that we can make them private later.
 
-Move the responsibility of fetching and storing data out of the analysis module
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Move the responsibility of fetching and storing data out of the analysis module [in progress]
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 As a first step towards further isolating the analysis pipeline of ERT and
 opening up for the possiblity of a stateless Python API for analysis we are to
 separate the responsibility of data fetching and storing (including the
 knowledge of EnKFMain and storage) and all logic for how analysis is done. In
 addition, we should improve the test base and seek local code improvements.
 
-Delete the deprecated interaction with the queue system
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-After production setting the ensemble evaluator, we should phase out the
-possibility of interacting with the queue system bypassing the legacy
-evaluator.
+Data record maturation [in progress]
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Extend the functionality of the records to be able to carry blob data,
+hierarchical data (including summary data), support transformations and
+serialization of record data.
 
-Run ERT2 forward models using prefect evaluator
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Transpile the ERT2 forward model into a single Unix Step that can be executed
-by the prefect evaluator. This should be introduced via a feature flag, that is
-later made default, before the legacy evaluator is removed.
+Act upon feedback on new data visualiser
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Actively seek feedback on webviz-ert and act upon the feedback such that
```

Comment:
Does not really belong here perhaps, but should we add a word about `everviz`?

Reply:
I suggest we do it when start the milestone instead :) This is not the scout milestones, it is the ert milestones...